cli/export_keys_from_mnemonic.go

```diff
@@ -19,7 +17,7 @@ var exportKeysCmd = &cobra.Command{
 	Use:   "export-keys",
 	Short: "exports private/public keys based on given mnemonic",
 	Run: func(cmd *cobra.Command, args []string) {
-		logger := logging.Build("export-keys", zap.DPanicLevel, nil)
+		logger := zap.L()
```

Comment:
need to set global logger here & name the logger
```suggestion
		logger := zap.L().Named("export-keys")
```